packages/kcmsf/src/pages/result.tsx

```diff
@@ -26,35 +28,37 @@ export const Result = () => {
 
   const preMatches = useMemo(
     () =>
-      matches?.pre.filter((match) => match.departmentType === department) ?? [],
-    [matches, department]
+      matches?.pre.filter((match) => match.departmentType === departmentType) ??
+      [],
+    [matches, departmentType]
   );
 
   const mainMatches = useMemo(
     () =>
-      matches?.main.filter((match) => match.departmentType === department) ??
-      [],
-    [matches, department]
+      matches?.main.filter(
+        (match) => match.departmentType === departmentType
+      ) ?? [],
+    [matches, departmentType]
   );
 
   return (
-    <>
-      <Select
-        label="部門"
-        data={config.departments.map((element) => ({
-          value: element.type,
-          label: element.name,
-        }))}
-        value={department}
-        defaultValue={config.departments[0].type}
-        onChange={(value) => value && setDepartment(value as DepartmentType)}
-      />
-      <Flex direction="column" gap={20}>
-        <Title order={3}>{config.department[department].name}</Title>
-        <MainResultTable matches={mainMatches} teamNames={teamNames} />
-        <PreResultTable matches={preMatches} />
-      </Flex>
-    </>
+    <Flex
+      direction="column"
+      align="center"
+      justify="center"
+      gap="{20}"
+      mb={"2rem"}
```

Comment:
```suggestion
      direction="column"
      align="center"
      gap={20}
```
プロパティの指定ミスと不要なスタイルがあるのでそこだけお願いします！